tests/toolchain/build-pass/print_include_dirs.json

```diff
@@ -0,0 +1,11 @@
+{
+   "tests" : [
+     {
+       "compile_flags": ["--eosio-include-dirs"],
```

Comment:
Could some include paths, e.g. `-I/tmp` and `-I /tmp`, be added in `compile_flags`, and check if the expected output contains such paths? (Similarly, for the below test for `--eosio-link-dirs`) All other parts look great.